### PR TITLE
Porting Rosetta problems

### PR DESCRIPTION
**Pull request by NitronR**
#### Pre-Submission Checklist

- [ ] Your pull request targets the `dev` branch.
- [ ] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/challenge-tests`)
- [ ] All new and existing tests pass the command `npm test`.
- [ ] Use `npm run commit` to generate a conventional commit message.
    Learn more here: <https://conventionalcommits.org/#why-use-conventional-commits>
- [ ] The changes were done locally on your machine and NOT GitHub web interface.
    If they were done on the web interface you have ensured that you are creating conventional commit messages.

#### Checklist:

- [ ] Tested changes locally.
- [ ] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes #XXXXX

### CODE REVIEW

**Comment by scissorsneedfoodtoo** [comment]
Thank you for the ports, @NitronR. I just noticed a couple of things with the descriptions for some challenges.

File: challenges/08-coding-interview-prep/rosetta-code.json [hunk 1]
```diff
@@ -7372,6 +7372,395 @@
...
+            "}"
+          ],
+          "head": [],
+          "tail": []
+        }
+      }
+    },
+    {
+      "title": "Long multiplication",
+      "description": [
```
Comment (scissorsneedfoodtoo) [inline challenges/08-coding-interview-prep/rosetta-code.json]:
```
"description": [
        "Explicitly implement  <a href=\"https://en.wikipedia.org/wiki/long multiplication\">long multiplication</a>.",
        "This is one possible approach to arbitrary-precision integer algebra.",
        "Write a function that takes two strings of large numbers as parameters. Your function should return the product of these two large numbers as a string.",
        "Note: In JavaScript, arithmetic operations are inaccurate with large numbers, so you will have to implement precise multiplication yourself."
      ],
```

File: challenges/08-coding-interview-prep/rosetta-code.json [hunk 2]
```diff
@@ -7372,6 +7372,395 @@
...
+          "name": "index",
+          "contents": ["function lcs (a, b) {", "  // Good luck!", "}"],
+          "head": [],
+          "tail": []
+        }
+      }
+    },
+    {
+      "title": "Longest increasing subsequence",
+      "description": [
```
Comment (scissorsneedfoodtoo) [inline challenges/08-coding-interview-prep/rosetta-code.json]:
```
      "description": [
        "Calculate and show here a <a href=\"https://en.wikipedia.org/wiki/Longest increasing subsequence\">longest increasing subsequence</a> of the list:",
        "$\\{3, 2, 6, 4, 5, 1\\}$",
        "And of the list:",
        "$\\{0, 8, 4, 12, 2, 10, 6, 14, 1, 9, 5, 13, 3, 11, 7, 15\\}$",
        "Write a function that takes an array of numbers as a parameter and returns the longest increasing subsequence."
      ],
```

**Comment by NitronR** [comment]
@scissorsneedfoodtoo Changes done.

**Pull request merged**